Makefile

```diff
@@ -512,6 +520,14 @@ $(DEB_PKG_CHANGELOG_FILE):
 prod-deb-package: $(DEB_PKG_CHANGELOG_FILE)
 	debuild --preserve-envvar DEB_SKIP_DH_AUTO_CLEAN -b -uc -us
 
+$(PLANTUML_JAR):
+	curl -fsS --create-dirs -o $@ https://netcologne.dl.sourceforge.net/project/plantuml/${PLANTUML_V}/plantuml.${PLANTUML_V}.jar
+
+build-uml: ${PLANTUML_JAR} ${uml-svg-files}
```

Comment:
Would it be possible to hook this (puml -> svg) to CI so to prevent it not to be working in the future and go unnoticed?

Reply:
Sure, we could do that.